frontend/src/features/projects/list/properties/columns.tsx

```diff
@@ -1,9 +1,8 @@
-import BuildingSvg from 'assets/images/icon-business.svg';
-import LandSvg from 'assets/images/icon-lot.svg';
+import { ReactComponent as BuildingSvg } from 'assets/images/icon-business.svg';
```

Comment:
Doing it this way allows us to override styling.  If we use `<img src={BuildingSvg}/>` then you can't.